src/app/frontend/deploy/createsecret.scss

```diff
@@ -14,6 +14,10 @@
 
 @import '../variables';
 
+.kd-create-secret-dialog {
+  min-width: 600px;
```

Comment:
Regarding both of the `width` styles: MS-Edge and Chrome seem to have different behavior when it comes to the size of DOM elements. Chrome makes them wider, while Edge as narrow as possible. So leaving the `width` unset (e.g. in the two cases above) leads to very different representations in the two browsers (and in Edge it looks terrible #nosurprise).